index.bs

```diff
@@ -47,21 +51,39 @@ Concepts {#concept}
 Crash {#concept-crash}
 -----
 
+Any time a process controlling [=Documents=] is abruptly terminated, whether by
+a supervising process within the user agent, or by the underlying operating
+system, the process is said to have <dfn lt="crash">crashed</dfn>. This may be
+due to a logic error in the browser, an unexpected hardware fault, the computer
+running out of needed resources, among other reasons. Specific kinds of
+[=crashes=] are detailed below.
+
 Out-of-Memory {#concept-oom}
 -------------
 
+Memory in computing systems is a finite resource, and is generally granted to
+running processes on request. When a request to allocate additional memory is
+denied, this may be treated as an unrecoverable error, and the requesting
+process may be terminated. A process terminated for this reason has [=crashed=]
+due to an <dfn>out-of-memory condition</dfn>.
+
 Unresponsive {#concept-unresponsive}
 ------------
 
+A process may be terminated in order to destroy a page which is no longer able
+to respond to user input. This may happen, for instance, because of an infinite
+loop in a script in the page, such that the script never completes and will
+never return control back to the event loop. A process terminated for this
+reason has [=crashed=] due to an <dfn>unresponsive condition</dfn>.
 
 Crash Reports {#crash-report}
 =============
 
 <dfn>Crash reports</dfn> indicate that the user was unable to continue using the
 page because the browser (or one of its processes necessary for the page)
-crashed. For security reasons, no details of the crash are communicated except
-for a unique identifier (which can be interpreted by the browser vendor), and
-optionally the reason for the crash (such as "oom").
+[=crashed=]. For security reasons, no details of the crash are communicated
```

Comment:
```suggestion
[=crashed=]. Crash reports aren't sent for every crash, for instance, if the [=coordinator process=] crashes. For security reasons, no details of the crash are communicated
```

Adding a bit about crashes being best effort, and there are cases where they won't be sent. I use coordinator process as the example because it's well defined and obvious, but GPU might be better in terms of user impact?